src/applications/vaos/new-appointment/components/ReviewPage/index.jsx

```diff
@@ -133,33 +124,3 @@ export function ReviewPage({
     </div>
   );
 }
-
-ReviewPage.propTypes = {
```

Comment:
It looks like that method can be used to check types outside of React props. So it would kind of be recreating the PropTypes pattern for our hooks setup.